src/Events/Integrations/Plugins/Event_Tickets/Provider.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace TEC\Events\Integrations\Plugins\Event_Tickets;
+
+use TEC\Common\Integrations\Traits\Plugin_Integration;
+use TEC\Events\Integrations\Integration_Abstract;
+
+/**
+ * Class Provider
+ *
+ * @since   TBD
+ *
+ * @package TEC\Events\Integrations\Plugins\Event_Tickets
+ */
+class Provider extends Integration_Abstract {
+	use Plugin_Integration;
+
+	/**
+	 * @inheritDoc
+	 */
+	public static function get_slug(): string {
+		return 'event-tickets';
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public function load_conditionals(): bool {
+		return function_exists( 'tribe_tickets' );
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	protected function load(): void {
+		// Loads Tickets Emails.
+		$this->container->register( \TEC\Events\Integrations\Modules\Ticket_Emails\Provider::class );
```

Comment:
@bordoni I think it really doesn't make sense to have a folder for modules. The functionality that we're integrating pretty much belongs to Event Tickets, and it makes way more sense to have it in the `Event_Tickets` folder. 

As a dev, if I would be looking for an Event Tickets integration I'd definitely look in the `Event_Tickets` folder rather than in modules. In this case the email integration it's bigger than other integrations that we would have, so it's clearer that it needs some more "organization". But what happens when we need to integrate something really simple for Event Tickets? We'll have one functionality living in `Modules` and another one in `Plugins`. 

I think this is a clear example about how we over-engineer things sometimes and it's really unclear.

In the case we decide to continue with this organization (which in my opinion is wrong), we'll need to update the namespace and folder to `Tickets_Emails`, both in plural.